husky_services/logger.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define(function() {
+
+    'use strict';
+
+    var instance = null,
+        getInstance = function() {
+            if (instance == null) {
+                instance = new Logger();
+            }
+            return instance;
+        };
+
+    function Logger() {
+    }
+
+    Logger.prototype = window.Husky.logger;
```

Comment:
How is the `logger` injected in Husky? Is there no better way than retrieving it from the `window` variable?